.rhcicd/pr_check.sh

```diff
@@ -19,7 +19,7 @@ REF_ENV="insights-stage"
 COMPONENTS_W_RESOURCES="pulp"
 
 # Only deploy one small red hat repo
-EXTRA_DEPLOY_ARGS="--set-parameter content-sources-backend/OPTIONS_REPOSITORY_IMPORT_FILTER=small"
+EXTRA_DEPLOY_ARGS="--set-parameter content-sources-backend/OPTIONS_REPOSITORY_IMPORT_FILTER=small --set-parameter 'content-sources-backend/NIGHTLY_CRON_JOB=5 4 25 12'"
```

Comment:
```suggestion
EXTRA_DEPLOY_ARGS="--set-parameter content-sources-backend/OPTIONS_REPOSITORY_IMPORT_FILTER=small --set-parameter 'content-sources-backend/NIGHTLY_CRON_JOB=5 4 25 12'"
```
Getting a stray backslash in the test run. Maybe just quote the numbers?

Reply:
sorry, it two backslashes, so its escaping:
`-set-parameter ''\''content-sources-backend/NIGHTLY_CRON_JOB=5' 4 25 '12'\'''`